lib/GenomeFileUtil/core/GenomeInterface.py

```diff
@@ -429,16 +429,33 @@ def sizeof_fmt(num):
         if g['taxon_ref'] == "ReferenceTaxons/unknown_taxon":
             warnings.append('Unable to determine organism taxonomy')
 
-        print("Subobject Sizes:")
-        for x in ('cdss', 'mrnas', 'features', 'non_coding_features', 'ontologies_present'):
+        # MAX_GENOME_SIZE = 300000000 # UNCOMMENT TO TEST FAILURE MODE. Set to size needed
```

Comment:
Mocks are the best way to do this. They can be kind of a complicated to get right so I'll try pushing a working example to this branch for you to look at and tweak as you see fit

Reply:
I forgot I already added in a test for this function here: https://github.com/kbaseapps/GenomeFileUtil/blob/master/test/genome_utils_test.py#L63

You should extend this test to encompass your new logic using it's mock